### ISSUE TITLE
[BUG] No status code is printed when `--parallel` is used 

### ISSUE THREAD

**Issue by rew1nter** [epi052/feroxbuster#1010]
**Describe the bug**
Even when used with verbose flag. Status codes are not shown in parallel mode. Only raw http URLs are printed. I also had weird URL characters mess up terminal a few times.

**To Reproduce**
1. Run with parallel flag and stdin input 

**Expected behavior**
I wanted to see status codes when running and not have HTTP URL characters mess up the terminal
 
**Environment (please complete the following information):**
 - feroxbuster version: 2.10.0
 - OS Arch linux

**Comment by rew1nter** [comment]
just read the docs. I think it should rather be a feature request

**Comment by epi052** [comment]
give the build [here](https://github.com/epi052/feroxbuster/actions/runs/7886390569) a try and let me know how it goes.

you're forced to select from --silent or --quiet when using --parallel

**Comment by rew1nter** [comment]
Alright man. Thank you for your effort

**Comment by epi052** [comment]
merged into main, will push a new release soon

**Comment by epi052** [comment]
@all-contributors add @rew1nter for idea

**Comment by rew1nter** [comment]
Sweet!
